openpower/package/openpower-pnor/openpower-pnor.mk

```diff
@@ -74,8 +74,7 @@ define OPENPOWER_PNOR_INSTALL_IMAGES_CMDS
             -wink_binary_filename $(BR2_HOSTBOOT_BINARY_WINK_FILENAME) \
             -occ_binary_filename $(OCC_STAGING_DIR)/$(BR2_OCC_BIN_FILENAME) \
             -targeting_binary_filename $(BR2_OPENPOWER_TARGETING_ECC_FILENAME) \
-            -openpower_version_filename $(OPENPOWER_PNOR_VERSION_FILE) \
-            -xz_compression $(BR2_OPENPOWER_PNOR_XZ_ENABLED)
+            -openpower_version_filename $(OPENPOWER_PNOR_VERSION_FILE)
```

Comment:
Can you also do an update to the PNOR package version?  I merged your corresponding pull request from pnor.